README.md

```diff
@@ -109,6 +110,7 @@ pip install -e ".[dev]"
 * **link** Define links between nodes.
 * **units** A statement introducing NeXus-compliant NXDL units arguments, like NX_VOLTAGE
 * **dimensions** Details which dimensional arrays to expect
+* **dim** Shorthand notation for dimensions, e.g., `(n, )` for an 1D array of length `n` or `(n, m)` for an 2D array of size `n x m`.
```

Comment:
() for a scalar but this must not be written because in NeXus scalar values must not be detailed with dimensions/dim keyword

Reply:
Indeed, I think it would just confusion + I believe that it's better to not specify it in this case.